test/e2e/apps/network_partition.go

```diff
@@ -30,14 +30,15 @@ import (
 
 	"k8s.io/client-go/tools/cache"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
this change isn't needed (same below)

Reply:
This was added automatically be `goreturns` when saving files.. will remove it.

Reply:
Fixed